proxima-chapter-1.md

```diff
@@ -146,21 +146,23 @@ We need to choose an Availability Domain, a Compute shape (4, 8, or 16 CPU - 4 w
 
 Once the instance has been created, let's open a Secure Shell to the instance via the assigned Public IP seen in the Compute Instance page:**
 
-`ssh -i <Your Private Key>pc@<Public IP>
+```bash
+$ ssh -i <Your Private Key>pc@<Public IP>
 Last login: never (ever)
 -bash: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such
 file or directory
-[opc@ggsa ~]$ ls
+$ ls
 README.txt
-[opc@ggsa ~]$`
+```
```

Comment:
Any time a piece of code contains multiple lines, use a fenced code block, optionally (preferably) with a language specifier (bash, python, rust, etc.). Shell commands like this can use `bash` or `sh`.

When showing a command line, use `$` to signify the prompt, removing any custom prompt styling/working directory.